views/postgres.ejs

```diff
@@ -65,6 +68,39 @@ database: ${username}</code>`
       }
     })
   }
+  const anon_createdb = () => {
+    if (disabled) return
+    disabled = true
+    const button = document.getElementById("anon_createdb_button")
+    button.classList.add("disabled")
+    button.innerText = "Creating..."
+    fetch("/api/anonymous/Postgres", { method: "POST" }).then((r) => r.json()).then((r) => {
+      console.log(r)
+      if (r.error) {
+        const message = res.error.message;
+        alert(message);
+        disabled = false
+        button.classList.remove("disabled")
+        button.innerText = "Create my database"
+      } else {
+        credentials.innerHTML = `<code>host: learndatabases.dev 
```

Comment:
Why is this different from createdb function? Is it possible to use the introduction.append function?
Also, instead of 2 different createdb functions, I think it would make sense to combine them and create just 1 createdb function. And then the back-end can check whether to make a `anon` or `non-anon` 

Or another idea I can think of that is better, is to have the fetch decide which URL to hit. Depending on whether the user is logged in or not.

Reply:
That's absolutely true. Even the API can be integrated. So I changed it again. Thanks for feedback